addons/event/report/event_event_templates.xml

```diff
@@ -215,6 +195,22 @@
         <t t-call="web.html_container">
             <t t-set="event" t-value="event._set_tz_context()"/>
             <t t-set="main_object" t-value="event"/>
+            <t t-set="simplified" t-value="False"/>
```

Comment:
Don't like "simplified", as it does not really means something ... what is a "simplified" ticket ?

Reply:
responsive_html then ?

for the templates, if possible always better to have the same naming :)